src/Mod/TechDraw/App/CosmeticExtension.cpp

```diff
@@ -42,7 +42,7 @@ CosmeticExtension::CosmeticExtension()
     static const char *cgroup = "Cosmetics";
 
     EXTENSION_ADD_PROPERTY_TYPE(CosmeticVertexes, (nullptr), cgroup, App::Prop_Output, "CosmeticVertex Save/Restore");
-    EXTENSION_ADD_PROPERTY_TYPE(CosmeticEdges, (nullptr), cgroup, App::Prop_Output, "CosmeticEdge Save/Restore");
```

Comment:
This property needs to be retained for backwards compatibility.  We can stop using it for new documents.
- need to modify the Restore method of PropertyCosmeticEdgeList to create the equivalent "Cosmetic" and update PropertyCosmeticList
    - or maybe add a call in onDocumentRestored/onExtendedDocumentRestored to move any CosmeticEdges in PropertyCosmeticEdgeList to PropertyCosmeticList and clear PropertyCosmeticEdgeList

Reply:
@WandererFan Do you know from where PropertyCosmeticList is called?

Reply:
PropertyCosmeticList is your new property?  You would have to search for Cosmetics.getValues() and Cosmetics.setValues.